modules/KIWILinuxRC.sh

```diff
@@ -6967,6 +6967,21 @@ function mountSystem {
     return $retval
 }
 #======================================
+# mountLivecd
+#--------------------------------------
+function mountLivecd {
+    if [ -n $TORAM ]; then
+        mkdir -p ${LIVECD}R $LIVECD && eval mount $cdopt $biosBootDevice ${LIVECD}R
+        SIZE="$(/usr/bin/du -s /${LIVECD}R | /usr/bin/gawk '{print int($1*1.1)}')"
+        test -n $SIZE || SIZE="800000"
```

Comment:
I think it can't happen that SIZE is empty and if it happens we should log this and proceed with the normal way of mounting the CD